xml/assets/.gitignore

```diff
@@ -0,0 +1,7 @@
+# The assets directory is for caching only.
+# https://stackoverflow.com/a/932982/7660197
+
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
It's for caching *between* runs of `xml/regression.sh`, so a tempdir through something like `mktemp` wouldn't work easily. How much difference that caching makes is questionable, but it's already in the script.

Could put a temp dir with a hard-coded path into /tmp, or keep `xml/assets` but don't check the empty directory into the source tree (would require some additional existence checks but not a big problem).

Reply:
Hmm, caching probably isn't worth the complexity then. This should eventually be run by CI, which always starts clean anyway. Alternatively, if we can verify the license is acceptable, we can just add the relevant files to the repository.

Reply:
> license

There are only two files that we download and cache:
* https://github.com/relaxng/relaxng.org/blob/master/relaxng.rng
  * There doesn't seem to be a license.
* https://github.com/relaxng/jing-trang/blob/master/eg/xslt.rng
  * Closest thing to a license seems to be the following: https://github.com/relaxng/jing-trang/blob/master/copying.txt

Reply:
>     * https://github.com/relaxng/relaxng.org/blob/master/relaxng.rng
>       
>       * There doesn't seem to be a license.

It's part of the RNG spec https://relaxng.org/spec-20011203.html ("This document and translations of it may be ...")

>     * https://github.com/relaxng/jing-trang/blob/master/eg/xslt.rng
>       
>       * Closest thing to a license seems to be the following: https://github.com/relaxng/jing-trang/blob/master/copying.txt

Bah, I don't want to have to add either license to Pacemaker's collection (and jump through hoops to get distro approval). Let's just download them :-/